### ISSUE TITLE
training data json file

### ISSUE THREAD

**Issue by yscoffee** [ari9dam/MathStudent#1]
Hi

Could you provide the annotated json file as shown in the following line ?
https://github.com/ari9dam/MathStudent/blob/master/src/main/java/nlp/app/math/ThreeCrossValidation.java#L36

we want to do retraining.

**Comment by ari9dam** [comment]
Hi ,

The dataset is added. Currently the convergence condition checks if the gradient norm is below 0.00001. But that takes time. You can stop the parameter update after 1000 iterations and you should get the same accuracy. Also I have updated the readme. Please see the Note section.

Thank you,
Arindam

**Comment by yscoffee** [comment]
many thanks
